app/lib/scc_manager.rb

```diff
@@ -10,7 +10,7 @@ def self.get_scc_data(base_url, rest_url, login, password)
     end
 
     url = base_url + rest_url
-    auth_header = { Authorization: 'Basic ' + Base64.encode64("#{login}:#{password}").chomp,
+    auth_header = { Authorization: format('Basic %s', Base64.encode64("#{login}:#{password}").chomp),
```

Comment:
Is there a special reason to use `format()` here?

Otherwise, I'd suggest to use `"#{}"`, which we use everywhere else.

Reply:
Can you actually nest `"#{}"`?